apps/client/src/views/cuesheet/cuesheet-table/cuesheet-table-elements/cuesheetCols.tsx

```diff
@@ -27,18 +28,24 @@ function MakeTimer({ getValue, row: { original } }: CellContext<OntimeRundownEnt
   );
 }
 
-function MakeDuration({ getValue }: CellContext<OntimeRundownEntry, unknown>) {
-  const { hideTableSeconds } = useCuesheetOptions();
-  const cellValue = (getValue() as number | null) ?? 0;
+function MakeDuration({ getValue, row, table }: CellContext<OntimeRundownEntry, unknown>) {
```

Comment:
In `MakeTimer` moving the `DelayIndicator` to after the timer would look better i think

Reply:
Do you have an opinion on the delay format here?

> When an event contains delays, what should be displayed
- a) both the scheduled and delayed times in the table and the delay amount in the tooltip? (ie: "+10min")
- b) the scheduled time in the table and the new delayed time in the tooltip? (ie: "+10 min, new start 10:10:00")
- c) the delayed time in the table and the scheduled time in the tooltip? (ie: "+10min, scheduled start 10:00:00")

I would personally be inclined towards b)